inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/KBStatement.java

```diff
@@ -130,64 +148,46 @@ public void setLanguage(String aLanguage)
     {
         language = aLanguage;
     }
-    
+
     public boolean isInferred()
     {
         return inferred;
     }
 
-    public void setInferred(boolean aInferred)
+    public void setInferred(boolean isInferred)
```

Comment:
`aInferred` - "a" stands for "argument". All arguments should start with "a" to distinguish them from fields and to avoid "this".